floris/wind_data.py

```diff
@@ -368,6 +376,20 @@ def resample(self, wd_step=None, ws_step=None, inplace=False):
             - Uses the bin weights feature in TimeSeries to resample the wind rose.
             - If `ws_step` or `wd_step` is not specified, it uses the current values.
         """
+
+        # If ws_step is passed in, confirm is it at least as large as the current step
+        if ws_step is not None:
+            if len(self.wind_speeds) >= 2:
+                if ws_step < self.wind_speeds[1] - self.wind_speeds[0]:
+                    raise ValueError("ws_step must be at least as large as the current step")
```

Comment:
@paulf81 maybe here, we should print what the current step is, which might help them set it correctly? 
something like:
```python
current_ws_step = self.wind_speeds[1] - self.wind_speeds[0]
if ws_step < current_ws_step:
    raise ValueError("ws_step provided must be at least as large as the current ws_step ({0:.2f} m/s)".format(current_ws_step)
```

Reply:
Sounds good, making the change...